### ISSUE TITLE
Add Veteran facing alert to notify them of missing letters

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#37032]
## Issue Description
As discovered in #36788, letters are not being sent from VBMS after December 9, 2021. As a result Veterans are not able to download any letters after that date until the problem has been resolved. DMC has requested that we add an alert to notify Veterans who may be looking to download a letter that was sent after December 9th. This issue is for design to mock up that alert. 

---
## Tasks
- [x] Mock up design for Veteran facing pop up alert
- [x] Share with stakeholders for review

## Acceptance Criteria
- [x] Pop up is ready for FE development.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by rileyorr** [comment]
@denisecoveyduc the wonderful @troyriser helped me combine both the existing alert about repayment plans with the new issue of unavailable downloadable letters. What we came up with can be seen below: 

![Screen Shot 2022-02-24 at 1.32.08 PM.png](https://images.zenhubusercontent.com/5d66a8f72a515f013c440719/d8c7e450-e37c-4a58-8d46-6045dfbbc791)

[I've also added the alert to a mocked up debt letters page](https://preview.uxpin.com/067dc9cad4c5966bc8a5f3a400cf10206e6a51d4?_gl=1*p5ij5u*_gcl_aw*R0NMLjE2NDM3MzgwNDMuRUFJYUlRb2JDaE1JcElULW5vbmY5UUlWTXRTekNoMk9aQUM4RUFBWUFTQUFFZ0tHQnZEX0J3RQ..#/pages/148133698/simulate/no-panels?mode=i). The alert has been sent to Jill's team for approval.
